File: scripts/SecretsTest3.java
// SecretsTest3.java
public class SecretsTest3 {
    // Intentionally exposing a hardcoded password
    private static final String PASSWORD = "mySecretP@ssw0rd"; // Example password

    public static void main(String[] args) {
        login();
    }

    private static void login() {
        System.out.println("Logging in with password: " + PASSWORD);
    }
}

File: scripts/secrets_test_3.py
# secrets_test_3.py
import os

# Intentionally exposing a secret stored in an environment variable
DATABASE_URL = os.getenv("DATABASE_URL", "postgres://user:password@localhost/db")  # Example URL

def connect_to_database():
    print(f"Connecting to database at: {DATABASE_URL}")

if __name__ == "__main__":
    connect_to_database()
